TerminalDocs/customize-settings/profile-settings.md

```diff
@@ -171,6 +171,21 @@ This sets the profile's font size in points.
 
 **Default value:** `12`
 
+### Font weight ([Preview](https://aka.ms/terminal-preview/))
+
+This sets the weight (lightness or heaviness of the strokes) for the profile's font.
+
+**Property name:** `fontWeight`
+
+**Necessity:** Optional
+
+**Accepts:** `"normal"`, `"thin"`, `"extra-light"`, `"light"`, `"semi-light"`, `"medium"`, `"semi-bold"`, `"bold"`, `"extra-bold"`, `"black"`, `"extra-black"`, or an integer corresponding to the numeric representation of the OpenType font weight
+
+**Default value:** `"normal"`
```

Comment:
Are you planning on including examples?

Reply:
I wasn't planning on it until weights come in for Cascadia?

Reply:
I mean, there _are_ other fonts that exist :) haha